<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>82</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="CStateButton" name="btnSystem">
       <property name="text">
        <string>System</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CStateButton" name="btnWork">
       <property name="text">
        <string>Work</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CStateButton" name="btnBelt">
       <property name="text">
        <string>Belt</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>9</y>
      <width>684</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="CStateButton" name="btnScd1">
       <property name="text">
        <string>Menu1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CStateButton" name="btnScd2">
       <property name="text">
        <string>Menu2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CStateButton" name="btnScd3">
       <property name="text">
        <string>Menu3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CStateButton" name="btnScd4">
       <property name="text">
        <string>Menu4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CStateButton" name="btnScd5">
       <property name="text">
        <string>Menu5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CStateButton" name="btnScd6">
       <property name="text">
        <string>Menu6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CStateButton" name="btnScd7">
       <property name="text">
        <string>Menu7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CStateButton" name="btnScd8">
       <property name="text">
        <string>Menu8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CStateButton</class>
   <extends>QPushButton</extends>
   <header>cwidgetbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
